### PR TITLE
Add bloat query timeout config. Update doc, postgres, and golang version

### PR DESCRIPTION
**Pull request by bksteiny**
### Added

- Environment variable to configure bloat query timeout
- CHANGELOD.md

### Changed

- Postgres version in docker-compose.yaml
- Golang to 1.15
- Updated documentation

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 language: go
 
 go:
-- "1.11.x"
-- "1.12.x"
+- "1.15.x"
```
Comment (jonstacks) [inline .travis.yml]: Can we also add "1.16.x" to this? Its now the most recent version of go

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+## [0.3.0] - 2021-07-03
+
+### Added
+
+- Environment variable to configure bloat query timeout
+- CHANGELOD.md
```
Comment (jonstacks) [inline CHANGELOG.md]:
```suggestion
- CHANGELOG.md
```

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12-alpine as builder
+FROM golang:1.15-alpine as builder
```
Comment (jonstacks) [inline Dockerfile]: Any reason not to go to version 1.16 here?

File: pkg/config/config.go [hunk 4]
```diff
@@ -78,7 +78,7 @@ func DBUser() string {
 }
 
 // FullVacuumTimeoutSeconds returns the number of seconds of how long pg-dba will
-// wait for an Analyze operation on the DB.
+// wait for an Full Vacuum operation on the DB.
```
Comment (jonstacks) [inline pkg/config/config.go:81]: 👍

File: pkg/dba/dba.go [hunk 5]
```diff
@@ -35,6 +37,8 @@ func NewDefaultOptions() Options {
 		AnalyzeTimeout: 10 * time.Minute,
 		PostAnalyze:    true,
 
+		BloatQueryTimeout: 5 * time.Minute,
```
Comment (jonstacks) [inline pkg/dba/dba.go]: Maybe default this to 30 seconds to keep it passive?

File: .travis.yml (hunk 1 above)
Comment (bksteiny) [reply .travis.yml]: Good call. Bumped to 1.16.x

File: CHANGELOG.md (hunk 2 above)
Comment (bksteiny) [reply CHANGELOG.md]: 🌵

File: Dockerfile (hunk 3 above)
Comment (bksteiny) [reply Dockerfile]: Nope, no reason 👍

File: pkg/dba/dba.go (hunk 5 above)
Comment (bksteiny) [reply pkg/dba/dba.go]: Good idea

**Review by jonstacks — APPROVED** [review]

**Pull request merged**